### PR TITLE
Rebase ian/future-maps on top of sujayakar/action-loop

### PR DESCRIPTION
**Pull request by sujayakar**
I also wired in the animations!

https://github.com/a16z-infra/ai-town/assets/5784949/972ef7dd-45b3-4ead-a119-acb604698566

### CODE REVIEW

**Comment by 61cygni** [comment]
Wohoo!!!

**Comment by sujayakar** [comment]
@61cygni the new maps and animations are fire, btw. really elevates the game ;)

File: src/editor/maps/gentle.js [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+{ x: 800, y: 320, w: 32, h: 96, layer: 1, sheet: "gentlewaterfall.json", animation: "pixels_large" },
+{ x: 832, y: 320, w: 32, h: 96, layer: 1, sheet: "gentlewaterfall.json", animation: "pixels_large" },
+{ x: 1664, y: 576, w: 208, h: 208, layer: 1, sheet: "windmill.json", animation: "pixels_large" },
+{ x: 1440, y: 768, w: 208, h: 208, layer: 1, sheet: "windmill.json", animation: "pixels_large" },
+{ x: 1120, y: 608, w: 208, h: 208, layer: 1, sheet: "windmill.json", animation: "pixels_large" },
+{ x: 736, y: 384, w: 32, h: 64, layer: 2, sheet: "gentlesplash.json", animation: "pixels_large" },
+{ x: 768, y: 384, w: 32, h: 64, layer: 2, sheet: "gentlesplash.json", animation: "pixels_large" },
+{ x: 800, y: 384, w: 32, h: 64, layer: 2, sheet: "gentlesplash.json", animation: "pixels_large" },
+{ x: 832, y: 384, w: 32, h: 64, layer: 2, sheet: "gentlesplash.json", animation: "pixels_large" },
+];
```
Comment (ianmacartney) [inline src/editor/maps/gentle.js:327]: I noticed these are missing `mapwidth` even though we're still referencing that in init. @61cygni is this from an older editor export?

**Review by ianmacartney — APPROVED** [review]

**Comment by 61cygni** [comment]
Yeah, it is. Good catch.

**Pull request merged**